sources/packages/web/src/views/aest/institution/Programs.vue

```diff
@@ -1,13 +1,216 @@
 <template>
   <content-group>
-    <p class="category-header-large color-blue">Programs</p>
+    <p class="category-header-large color-blue">
+      All Programs ({{ institutionProgramsSummary.programsCount }})
+      <v-row class="float-right">
+        <v-col>
+          <InputText
+            name="searchProgramName"
+            v-model="searchProgramName"
+            placeholder="Search Program Name"
+          />
+        </v-col>
+        <v-col>
+          <v-btn
+            color="primary"
+            @click="goToSearchProgramName(searchProgramName, $event)"
```

Comment:
guru,  I am confused here, how are you sending button event and taking rows?

Reply:
That's a nice question, changed it after discussion 👍